### DISCUSSION TITLE
115网盘突然变成只能显示目录内容，无法播放了

### DISCUSSION THREAD

**Discussion by clbin001** [AlistGo/alist#6568] [Q&A]
刚才发现115网盘突然变成只能显示目录内容，无法播放了。在Alist的访问界面上点击图片或视频后，显示以下报错信息，不知道有没有解决方法

failed link: failed get link: {"state":false,"msg":"因产品升级调整，暂时关闭，给你带来不便，敬请谅解","errno":0,"data":""}: unexpected error

**Comment by BlueSkyXN** [comment]
历史上的明天

**Comment by flansing** [comment]
> 刚才发现115网盘突然变成只能显示内容，无法播放了。在Alist的访问界面上点击图片或视频后，显示以下错误信息，不知道有没有解决办法
> 
> 链接失败：获取链接失败：{"state":false,"msg":"因产品升级调整，暂时关闭，给您带来不便，敬请谅解","errno":0,"data":""}：意外错误

量劫法力到来，上古有五大量劫：开天，龙汉，巫妖，封神，西游，今有TK量劫，你就知足吧

**Comment by erosaishen** [comment]
我发现是文件后缀名被改变了mkv变成了mkv1，mp4被变成了mp41，且无法修改后缀，也无法下载，成了废资源

**Comment by DeclanZhu** [comment]
不知道是不是限制了海外IP地址的下载功能

**Comment by clbin001** [comment]
问题消失了，一切都恢复正常，可以正常使用了。一进入中国时间6月5日，就一切正常了，原来上面的这个bug只是因为昨天是特殊的日子，而导致的。
